### DISCUSSION TITLE
why bare-metal option not in the documentation?

### DISCUSSION THREAD

**Discussion by mm3906078** [qdrant/qdrant#4246] [Q&A] [answered]
Hi guys, I hope you having a wonderful day

I want to use qdrant in my application, and I prefer to use it on a VM/bare metal because I think I should get better performance rather than container deployment. Why does nobody in the documentation care about this? Is it safe to deploy it on bare metal?

**Comment by timvisee** [comment, marked as answer]
Running on bare metal is totally fine. But, it's more cumbersome to manage and scale.

Note that containers are Linux kernel namespaces, not VMs. They may not be as slow as you think. Networking may have a slight hit though.
